lib/Controller.js

```diff
@@ -191,6 +202,9 @@ module.exports = (function(){
             else {
                 logger.info("[%s] Entry is no longer live, stopping it", noLongerLiveEntry);
                 return liveEntry.stop()
+                    .then(() => {
+                        return that.crossDCsStreamPersistence.addEntry(noLongerLiveEntry);
```

Comment:
we need to be carfull here, since we don't want exception from addEntry to block our flow,
remove the return